core/translate/schema.rs

```diff
@@ -767,11 +767,183 @@ pub fn translate_drop_table(
     let r7 = program.alloc_register();
     program.emit_null(r6, Some(r7));
 
-    //  3. TODO: Open an ephemeral table, and read over triggers from schema table into ephemeral table
-    //  Requires support via https://github.com/tursodatabase/limbo/pull/768
+    //  3. Open an ephemeral table, and read over entries from the schema table for entries of the table whose root page was moved in the destroy operation
```

Comment:
This is always max 1 row right?

Reply:
yes, i've updated the docs in 3a8b18c481bb2573e511cfcaa5c039182ec11424